package.json

```diff
@@ -43,5 +43,8 @@
   "engines": {
     "node": ">=16.0.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "delegated-events": "^1.1.2"
+  }
```

Comment:
It seems as though I incorrectly installed this package. I installed using `yarn add delegated-events` . Is there a preferred way of installing this?

Reply:
If you want to install something just for the `prairielearn` app, you can do either of the following from the root of the repository:

- `yarn workspace @prairielearn/prairielearn add delegated-events`
- `cd apps/prairielearn && yarn add delegated-events`

Reply:
Thanks!

Reply:
This should be resolved now in [79c8d21](https://github.com/PrairieLearn/PrairieLearn/pull/9286/commits/79c8d211ab90d919e9f5ad7e7c810370f8422d01).